1.8/usage/jobs/getting-started.md

```diff
@@ -0,0 +1,120 @@
+---
+post_title: Getting Started
+nav_title: Getting Started
+menu_order: 10
+---
+
+You can create and administer jobs in the DC/OS web interface, the DC/OS CLI, or via the API. You can add permissions to your job via an access control list (ACL). [Learn more](/1.8/administration/id-and-access-mgt/permissions/service-acls).
```

Comment:
I don't know what "schedule-only" job means.     You can create a schedule free job :)  or a job with a schedule.